pyne/cpp_material.pxd

```diff
@@ -53,9 +53,8 @@ cdef extern from "material.h" namespace "pyne":
         void from_hdf5(char *, char *, int) except +
         void from_hdf5(char *, char *, int, int) except +
 
-        void write_hdf5(char *, char *, char *) except +
-        void write_hdf5(char *, char *, char *, float) except +
-        void write_hdf5(char *, char *, char *, float, int) except +
+        void deprecated_write_hdf5(char *, char *, char *, float, int) except +
+        void write_hdf5(char *, char *, float, int) except +
```

Comment:
Here I decided to expose only the C++ method we are actually using on the python binding (for `write_hdf5()`)